src/mmrelay/cli.py

```
@@ -13,12 +13,17 @@
+
+# Import version from package
+from importlib.metadata import version as _get_version
 from typing import Any
 
 import yaml
 
-# Import version from package
-from mmrelay import __version__
+try:
+    __version__ = _get_version("mmrelay")
+except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)